packages/office-addin-unit-test/src/defaults.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
defaults.ts?  The file names seems a bit vague.

And why does the usageData object need it's own file?

Reply:
Yeah, I guess `default` is a bad name for `usageData` in general. All of our packages scripts currently use `default`, so I guess we keep it for now and flag this to be changed in another task.